index.html

```diff
@@ -0,0 +1,14 @@
+<!doctype html>
+
+<html lang="en">
+<head>
+    <meta charset="utf-8">
+    <title>HTML & CSS Training</title>
+    <meta name="description" content="training">
```

Comment:
All this lines still have 4 spaces tabulation, should be 2. Maybe you have it fine in your project but it shows wrong in github?